File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
        for (int i = 0; i < t.size() - p.size() +1; i++)
    {
        string num = "";
        for (int j = i; j < i+ p.size(); j++)
        {
            num += t[j];
        }
        if (stoll(num) <= stoll(p)) answer++;
    }
    return answer;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cs
using System;

public class Solution {
    public int solution(string t, string p) {
        int answer = 0;
        for(int i =0; i<t.Length-p.Length+1;i++)
        {
            string s = t.Substring(i,p.Length);
            answer += long.Parse(p) >= long.Parse(s) ? 1:0;
        }
        return answer;
    }
}